### PR TITLE
fix #4162 chore(project): remove rapid

### LINKED ISSUE #4162: Remove Rapid 💀
RIP Rapid, you served well 🙏

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-687)
┆Issue Number: EXP-687

### PR DESCRIPTION
**Pull request by jaredlockhart**
Because

* Nimbus has now reached feature parity including visualizations

This commit

* Removes all the rapid project files and db fields and everything

### CODE REVIEW

**Review by emtwo — APPROVED** [review]
lgtm in terms of visualization

**Review by jodyheavener — COMMENTED** [review]
You'll want to update the main README to remove the rapid build command under Semi-dockerized setup and the mention of rapid under Frontend

**Review by jodyheavener — APPROVED** [review]
I wasn't terribly familiar with Rapid to start with, so others may have more insight, but as far as I can tell everything still starts up and works correctly. Just that little update to the readme is all I've noted

**Comment by jaredlockhart** [comment]
@jodyheavener I *ALWAYS* forget to update README XD good catch ;D

File: app/experimenter/legacy-ui/core/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "react": "16.13.1",
     "react-bootstrap": "1.3.0",
     "react-dom": "16.13.1",
-    "react-select": "3.1.0"
+    "react-select": "^3.1.1"
   },
```
Comment (tiftran) [inline app/experimenter/legacy-ui/core/package.json:45]: just curious, is there context of why the version bump/changes for react-select? I remember something about react-select hell in slack....

**Review by tiftran — APPROVED** [review]

File: app/experimenter/legacy-ui/core/package.json (hunk 1 above)
Comment (jaredlockhart) [reply app/experimenter/legacy-ui/core/package.json:45]: Well I think this whole workspace thing is kinda wonky where you define the requirements per 'project', ie for core, rapid, visualization, nimbus-ui, but they share a lock file and they all get installed into one node modules folder, and I think they weirdly overlap somehow?  So when I removed the rapid ui, suddenly nimbus didn't have access to the @types/react-select, so I think that was installed for rapid but not nimbus-ui but nimbus-ui could still import it as if it were, so i just reinstalled it everywhere and that semed to clear it up.

**Pull request merged**